cleanup-oracle.sh

```diff
@@ -76,6 +113,10 @@ while true; do
     shift
 done
 
+[[ "$ORA_STAGING" == "" ]] && {
+    ORA_STAGING=$ORA_SWLIB_PATH
```

Comment:
Please quote; though I'd hope nobody would put a space in the pathname here :-)

Reply:
swlib path quoted before, I assume it should be okay

Reply:
--ora-swlib-path)
        ORA_SWLIB_PATH="$2"
        shift;
        ;;